README.md

```diff
@@ -2,41 +2,47 @@ This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next
 
 ## Getting Started
 
-### Initial setup
+### Initial setup and subsequent updates
 
 1. Install nvm if you don't have it yet.
-2. Activate the right node version
+2. Activate the right node version (you can do all steps from 2 to 5 to make sure that the update does not fail).
 
-   ```bash
-   nvm use
-   ```
+```bash
+nvm use
+```
 
 3. Make sure the npm version is controlled with corepack:
 
 ```bash
 corepack enable npm
 ```
 
-3. If you need access to the Kausal private themes, login to https://npm.kausal.tech
-   using your Google credentials, copy the line with `npm config set ...` from
-   the settings dialog, and execute it in your shell. Ensure the `@kausal` scope
-   is associated with that registry by running:
+4. If you need access to the Kausal private themes:
+
+4.1. remove the cookie(s) with npm.kausal.tech from your browser's cookie history.
+
+4.2. login to https://npm.kausal.tech
+      using your Google credentials, copy the line with `npm config set ...` from
+      the settings dialog, and execute it in your shell. Ensure the `@kausal` scope
+      is associated with that registry by running:
```

Comment:
Markdown doesn't support nested lists including the parent number, so instead we'll need to do this:

```suggestion
4. If you need access to the Kausal private themes:
    1. Remove the cookie(s) with npm.kausal.tech from your browser's cookie history.
    2. Log in to https://npm.kausal.tech
      using your Google credentials, copy the line with `npm config set ...` from
      the settings dialog, and execute it in your shell. Ensure the `@kausal` scope
      is associated with that registry by running:
```

Reply:
And by the way, you can commit suggestions directly within GitHub with the "Commit suggestion" button above, pretty handy